build/Targets/Imports.targets

```diff
@@ -138,6 +138,10 @@
     <DisableImplicitFrameworkReferences Condition="'$(DisableImplicitFrameworkReferences)' == ''" >true</DisableImplicitFrameworkReferences>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+      
+    <!-- Only generate our runtimeconfig.json files for net core apps. It's unnecessary in desktop projects
+         but gets included in lots of output items like VSIX. -->
+    <GenerateRuntimeConfigurationFiles Condition="'$(TargetFramework)' != 'netcoreapp1.1' AND '$(TargetFramework)' != 'netcoreapp2.0'">false</GenerateRuntimeConfigurationFiles>
```

Comment:
I'll add one in the cleanup change. 

https://github.com/dotnet/roslyn/issues/20526